examples/SampleUtils.h

```diff
@@ -49,6 +49,7 @@ class ExampleBase {
     std::string mChannelScheme = "RGB";
     std::vector<int32_t> mOutputShape;
     std::string mDevice = "default";
+    bool mFused = true;
```

Comment:
Should default `mFused` be false? It would align with the model tests setting.

Reply:
Yes, false will align with webnn-polyfill model tests. I want to set true as default here for better C++ example performance. Which one is better? Thanks! @huningxin

Reply:
it makes sense to use fused for example by default.